tcs-docs/TaxonConcept.ttl

```diff
@@ -0,0 +1,363 @@
+    rdfs:comment "The host taxon in this relationship." ;
+    rdfs:domain :Relationship ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range :TaxonConcept .
+
+:hasInformation a owl:ObjectProperty ;
+    rdfs:label "Has Information" ;
+    rdfs:comment "A piece of information about this TaxonConcept." ;
+    rdfs:domain :TaxonConcept ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range <http://rs.tdwg.org/ontology/voc/SpeciesProfileModel#InfoItem> .
+
+:hasRelationship a owl:ObjectProperty ;
+    rdfs:label "hasRelationship" ;
+    rdfs:comment "A relationship to another TaxonConcept" ;
+    rdfs:domain :TaxonConcept ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range :Relationship .
+
+:primary a owl:DatatypeProperty ;
+    rdfs:label "Primary" ;
+    dc:relation :rank ;
+    rdfs:comment """If primary='true' the concept is the first level response to a query. If 'false'
+         the concept may be a secondary concept linked directly or indirectly to the definition of a
+         primary concept.""";
+    rdfs:domain :TaxonConcept ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range xsd:boolean .
+
+:relationshipCategory a owl:ObjectProperty ;
+    rdfs:label "relationshipCategory" ;
+    ns1:CommontcsEquivalence "/DataSet/TaxonConcepts/TaxonConcept/Relationships/Relationship/@type" ;
+    rdfs:comment "The kind of relationship this is." ;
+    rdfs:domain :Relationship ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range :TaxonRelationshipTerm .
+
+:toTaxon a owl:ObjectProperty ;
+    rdfs:label "toTaxon" ;
+    ns1:CommontcsEquivalence "/DataSet/TaxonConcepts/TaxonConcept/Relationships/Relationship/ToTaxonConcept" ;
+    rdfs:comment "The target taxon in this relationship." ;
+    rdfs:domain :Relationship ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range :TaxonConcept .
+
+:rankString a owl:DatatypeProperty ;
+    rdfs:label "Rank String" ;
+    dc:relation :rank ;
+    rdfs:comment """*Deprecated* The taxonomic rank of this concept as a string. A string
+         representation of the rank of this concept. It is highly recommended that the rank property
+         be used along with this one unless the correct rank is not available in the rank
+         vocabulary. No direct equivalent in TCS but is string version of
+         /DataSet/TaxonConcepts/TaxonConcept/Rank. Requested by data suppliers. After discussions
+         between a number of parties this property is considered to be deprecated. The rank of a
+         TaxonConcept should be indicated by including a TaxonName of the appropriate rank in the
+         hasName property even if the name object is empty apart from the rank. """;
+    rdfs:domain :TaxonConcept ;
+    rdfs:isDefinedBy ns1:TaxonConcept ;
+    rdfs:range xsd:string .
+
+:rank a owl:ObjectProperty ;
+    rdfs:label "Rank" ;
+    dc:relation :rankString ;
+    rdfs:comment """*Deprecated* The taxonomic rank of this concept. This is a link to an instance
```

Comment:
should we remove the deprecated properties for a discussion basis?

Reply:
I think it is best to leave this in for now, as TCS has rank under both TaxonName and TaxonConcept as well. I think the wrong one was deprecated.